src/home/actions.js

```diff
@@ -29,10 +29,12 @@ export const loadAnnouncements = () => async (dispatch, getState) => {
 }
 
 export const loadGroups = () => async (dispatch) => {
-    const { data } = await request("GET", "/groups/normal");
+    const userGroupsReq = await request("GET", "/groups/normal");
```

Comment:
Can instead be
`const { data: userGroups } = ...`
`const { data: otherGroups } = ...`